setup.py

```diff
@@ -28,23 +28,17 @@
         'Natural Language :: English',
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
         'Programming Language :: Python :: 3.11',
     ],
 
+    python_requires=">=3.7",
```

Comment:
Nice! I believe this makes the removal of older runtimes safe for users using py <37 as they will not get the new version, right?

Reply:
Yes, as long as they use a recent enough release of pip (v9.0 released 2016-11-02) and the environment that generates the package metadata uses setuptools v42 (released 2019-11-23) or later.